aten/src/ATen/native/native_functions.yaml

```diff
@@ -5361,14 +5361,21 @@
     CompositeExplicitAutograd: slice_backward
   autogen: slice_backward.out
 
+- func: slice_inverse(Tensor(a) self, Tensor src, int dim=0, SymInt? start=None, SymInt? end=None, SymInt step=1) -> Tensor(a)
```

Comment:
Could you add a note here, describing the situation + exactly when this op gets used? I **think** two invariants we want to uphold are:

(1) This op shouldn't really show up in eager mode (so e.g. XLA shouldn't have to implement it)

(2) This op shouldn't show up in a PT2 graph (so a PT2 backend shouldn't have to implement it)

And the only case where this really shows up is during fakeification; meaning that a **subclass** might have to implement it (but only if we have a subclass that is an input to the PT2 graph, and is a view of some other tensor)

Reply:
good call, added the note